charts/ccsm-helm/test/operate/configmap.golden.json

```diff
@@ -0,0 +1,47 @@
+---
```

Comment:
:question: Isn't this a YAML file? Is there a reason the golden files must be JSON? :thinking:

Reply:
no was by mistake :grin: